src/components/Dashboard/Details/Details.js

```diff
@@ -1,60 +1,135 @@
+}
+
+function getBeneficiary(addr) {
+  return (
+    <div className="item">
+      <Header sub>
+        <FormattedMessage id="details.beneficiary" />
+      </Header>
+      <Header onClick={() => copyToClipboard(addr)} style={{ cursor: "pointer" }}>
+        <img style={{ width: "13px", marginRight: "5px", paddingBottom: "5px" }} src={AddressIcon} />
+        <Popup
+          content={<FormattedMessage id="global.copied" />}
+          position="bottom center"
+          trigger={<span>{addressShortener(addr)}</span>}
+          on="click"
+        />
+        <Info message={<FormattedMessage id="helper.beneficiary" />} position="left center" />
+      </Header>
+    </div>
+  );
+}
+
+function getDate(id, date) {
+  const type = id.split(".");
+  return (
+    <div className={`item ${type[type.length - 1]}`}>
+      <Header sub>
+        <FormattedMessage id={id} />
+      </Header>
+      <Header>
+        <FormattedDate value={new Date(date * 1000)} year="numeric" month="long" day="numeric" />
+      </Header>
+    </div>
+  );
+}
+
+function getCliffPeriod(vestingCliff) {
+  return (
+    <div className="item">
+      <Header sub>
+        <FormattedMessage id="details.cliff_period" />
+      </Header>
+      <Header>
+        <FormattedMessage
+          id="details.cliff_period.time"
+          values={{
+            cliff: vestingCliff,
+            monthPl: (
+              <FormattedPlural
+                value={vestingCliff}
+                one={<FormattedMessage id="global.month" />}
+                other={<FormattedMessage id="global.month.plural" />}
+              />
+            ),
+          }}
+        />
+        <Info message={<FormattedMessage id="helper.cliff_period" />} position="left center" />
+      </Header>
+    </div>
+  );
+}
+
+function getAmount(id, value, symbol, helperId) {
+  return (
+    <div className="item">
+      <Header sub>
+        <FormattedMessage id={id} />
+      </Header>
+      <Header>
+        <FormattedNumber value={value} /> {symbol}
+        <Info message={<FormattedMessage id={helperId} />} position="left center" />
+      </Header>
+    </div>
+  );
+}
+
+function getRevocable(revocable) {
+  return (
+    <div className="item">
+      <Header sub>
+        <FormattedMessage id="details.revocable" />
+      </Header>
+      <Header>{revocable ? <FormattedMessage id="global.yes" /> : <FormattedMessage id="global.no" />}</Header>
+    </div>
+  );
+}
+
+function Details(props) {
+  const { contract, isBeneficiary, onRelease } = props;
+  const { symbol, released, balance, start, cliff, duration, releasableAmount, revocable } = contract;
+  const vestingCliff = getMonthDiff(start, cliff);
+  const total = balance + released;
+
+  const responsive = useResponsive();
+  const isMobile = responsive({ maxWidth: Responsive.onlyMobile.maxWidth });
+
+  return (
+    <div id="details" className={(isMobile && "mobile") || ""}>
+      {isMobile ? (
```

Comment:
This can be done with CSS. 
```
<div className="divToStyle">
  <div className="dates">
    {getDate("details.start", start)}
    {getDate("details.end", start + duration)}
  </div>
  {getBeneficiary(contract.beneficiary)}
</div>
````
You can reverse order of elements of "divToStyle" using flex and media queries.

Reply:
Done!